merged-packages/design-tokens/package.json

```diff
@@ -31,97 +36,32 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsup --clean && yarn build:types && yarn build:css",
-    "build-storybook": "storybook build",
-    "build:clean": "rimraf dist && yarn build",
+    "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references && yarn build:types && yarn build:css",
```

Comment:
Updating build to use ts-bridge and align with other packages, I assume types get build as well but keeping the css build